packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -157,5 +158,97 @@ describe('Rating', () => {
+                it('should display descriptive content', () => {
+                    // Arrange & Act
+                    const { validator } = VRating.props.ratingDisplayType;
+                    validator('short');
+
+                    propsData = {
+                        starRating: 2,
+                        maxStarRating: 5,
+                        ratingDisplayType: 'short'
+                    };
+                    wrapper = shallowMount(VRating, {
+                        propsData,
+                        localVue,
+                        i18n,
+                        mocks: {
+                            $tc
+                        }
+                    });
+
+                    const result = wrapper.find('[data-test-id="c-rating-displayType"]');
+
+                    // Assert
+                    expect(result.exists()).toBe(true);
+                });
+            });
+        });
+    });
+
+    describe('methods', () => {
+        describe('`getRatingDisplayFormat`', () => {
+            it('should exist', () => {
+                // Act & Assert
+                expect(wrapper.vm.getRatingDisplayFormat).toBeDefined();
+            });
+
+            describe('when invoked', () => {
+                describe('and `ratingDisplayType` is passed through', () => {
+                    it.each([
+                        ['noRating', 'No ratings yet'],
+                        ['short', '499'],
+                        ['medium', '2 of 5'],
+                        ['long', 'View 499 reviews']
+                    ])('should return the correct display type for each rating component', (type, expected) => {
+                        // Arrange
+                        propsData = {
+                            starRating: 2,
+                            totalReviews: 499,
+                            ratingDisplayType: type
+                        };
+
+                        wrapper = shallowMount(VRating, {
+                            propsData,
+                            localVue,
+                            i18n
+                        });
+
+                        // Act
+                        const result = wrapper.vm.getRatingDisplayFormat();
+
+                        // Assert
+                        expect(result).toBe(expected);
+                    });
+                });
+
+                describe('and `ratingDisplayType` is not passed through', () => {
+                    it('should return the default rating display type', () => {
+                        wrapper = shallowMount(VRating, {
+                            propsData,
+                            localVue,
+                            i18n
+                        });
+
+                        // Act
+                        const result = wrapper.vm.getRatingDisplayFormat();
+
+                        // Assert
+                        expect(result).toBe('No ratings yet');
```

Comment:
I will probably change this tomorrow because i think the default rating component should display just the stars with no text.

Reply:
Fixed.